chia/wallet/puzzles/load_clvm.py

```diff
@@ -1,53 +1,106 @@
 import importlib
 import inspect
 import os
+import time
+
+from typing import Optional, TextIO
 import pathlib
 
 import pkg_resources
-from clvm_tools.clvmc import compile_clvm as compile_clvm_py
 from chia.types.blockchain_format.program import Program, SerializedProgram
+from clvm_tools_rs import compile_clvm as compile_clvm_rust
+
+
+# Cribbed mostly from chia/daemon/server.py
+def create_exclusive_lock(lockfile) -> Optional[TextIO]:
+    """
+    Open a lockfile exclusively.
+    """
+
+    try:
+        fd = os.open(lockfile, os.O_CREAT | os.O_EXCL | os.O_RDWR)
```

Comment:
this looks very posix-y. does it work on windows?

Reply:
I will test it.